patches/roslyn-analyzers/0004-Giant-hacks.patch

```diff
@@ -0,0 +1,92 @@
+From 0cfe6de389cb1d8c730ae15a786c828d59974c04 Mon Sep 17 00:00:00 2001
+From: Omair Majid <omajid@redhat.com>
+Date: Mon, 9 Nov 2020 10:54:26 -0500
+Subject: [PATCH 4/5] Giant hacks
+
+---
+ eng/Versions.props                                        | 8 +++++---
+ ...icrosoft.CodeAnalysis.CSharp.BannedApiAnalyzers.csproj | 2 +-
+ ...oft.CodeAnalysis.VisualBasic.BannedApiAnalyzers.vbproj | 2 +-
+ .../GenerateGlobalAnalyzerConfigs.csproj                  | 4 +++-
+ src/Utilities/Compiler/Analyzer.Utilities.projitems       | 2 +-
+ 5 files changed, 11 insertions(+), 7 deletions(-)
+
+diff --git a/eng/Versions.props b/eng/Versions.props
+index da9b8d60a..c25eaf99d 100644
+--- a/eng/Versions.props
++++ b/eng/Versions.props
+@@ -24,14 +24,16 @@
+     <!-- Use the correct compiler version -->
+     <UsingToolMicrosoftNetCompilers>true</UsingToolMicrosoftNetCompilers>
+     <!-- Roslyn -->
+-    <MicrosoftCodeAnalysisVersion>3.8.0</MicrosoftCodeAnalysisVersion>
++    <!-- Source build doesn't produce the Microsoft.CodeAnalysis super-assembly ; use individual packages instead
++    <MicrosoftCodeAnalysisVersion>3.3.1</MicrosoftCodeAnalysisVersion>
++    -->
```

Comment:
"Metapackage" was the common term for this kind of package that has no content, only dependencies, when I was working with them in dotnet/runtime. And actually, since it doesn't have an assembly I'd expect it to be easy to add back. Filed https://github.com/dotnet/source-build/issues/1870 for this.